components/DirectoryOrgs.tsx

```diff
@@ -8,18 +8,12 @@ interface Props {
 }
 const OrgDirectoryCards: FC<Props> = ({ org }) => {
   return (
-    <div className="md:w-full h-auto flex flex-col justify-between items-center p-5 m-5 rounded-2xl">
-      <h4 className="text-lg font-bold text-center">{org.name}</h4>
-      <div className="w-full h-40 relative justify-center">
-        <Image
-          src={org.imageLink}
-          width="100%"
-          height="100%"
-          alt={org.name}
-          className="rectangle"
-        />
+    <div className="md:w-full h-auto flex flex-col justify-between p-5 m-5 rounded-2xl">
+      <h4 className="text-xl font-bold text-center md:text-left">{org.name}</h4>
+      <div className="w-auto h-40 relative justify-center rounded-sm p-2">
+        <Image src={org.imageLink} width={200} height={150} alt={org.name} />
```

Comment:
The images won't be squished because its parent component has an auto width, so it retains its correct size despite changes in window size.
<img width="1037" alt="image" src="https://user-images.githubusercontent.com/76502130/197109702-312b0263-8133-4003-b720-1b0c57ab181c.png">
<img width="929" alt="image" src="https://user-images.githubusercontent.com/76502130/197109780-ae06fbd7-7c96-445f-9ecc-cfadb8f11add.png">

Reply:
Thanks for clarifying. Maybe there is something going on on my end, though it appears as though the UTD logo has a lower height resulting in a slight oval shape

Reply:
> Thanks for clarifying. Maybe there is something going on on my end, though it appears as though the UTD logo has a lower height resulting in a slight oval shape

@RubenOlano Just to verify, is the current appearance of the UTD logo correct?

Reply:
Yes